sdks/unity/Tests/Runtime/PlayMode/AgonesSdkIntegrationTests.cs

```diff
@@ -0,0 +1,167 @@
+// Copyright 2024 Google LLC
+// All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using Agones;
+using System.Threading.Tasks;
+using System;
+
+namespace Tests.Runtime.Agones
+{
+    public class AgonesSdkIntegrationTests
+    {
+
+        private AgonesSdk sdk;
+        private AgonesBetaSdk betaSdk;
+        private AgonesAlphaSdk alphaSdk;
+
+        [UnitySetUp]
+        public IEnumerator UnitySetUp()
+        {
+            GameObject gameObject = new GameObject();
+            gameObject.AddComponent<AgonesSdk>();
```

Comment:
AddComponent() returns the component reference. If you wanted you could add the component and set the local class properties into one expression.